packages/@aws-cdk/aws-servicecatalog/README.md

```diff
@@ -186,6 +189,27 @@ If a misconfigured constraint is added, `synth` will fail with an error message.
 
 Read more at [Service Catalog Constraints](https://docs.aws.amazon.com/servicecatalog/latest/adminguide/constraints.html).
 
+### Tag update constraint
```

Comment:
While you're in this area, can you fix the heading for `### Tag Options` (it should be an H2, not an H3)?